Kai.BaseModule/KaiSDK.cs

```diff
@@ -44,7 +44,7 @@ public static partial class KaiSDK
 		/// <summary>
 		/// Contains the value of the default Kai that is connected to the SDK
 		/// </summary>
-		public static Kai DefaultKai { get; } = new Kai();
+		public static Kai DefaultKai { get; private set; } = new Kai();
```

Comment:
Since the `DefaultKai` is supposed to be a constant and should, ideally, never change, I'd suggest we have just a public getter and no setter
```suggestion
		public static Kai DefaultKai { get; } = new Kai();
```